tests/README.md

```diff
@@ -9,7 +9,7 @@ First of all, change directory to the root of `nebula-graph`, and build the whol
 
 ### Init environment
 
-Nebula Test framework depends on some thirdparty libraries, such as [nebula-python](https://github.com/vesoft-inc/nebula-python), [reformat-gherkin](https://github.com/OneContainer/reformat-gherkin), pytest, [pytest-bdd](https://pytest-bdd.readthedocs.io/en/latest/) and so on.
+Nebula Test framework depends on python3(>=3.8) and some thirdparty libraries, such as [nebula-python](https://github.com/vesoft-inc/nebula-python), [reformat-gherkin](https://github.com/OneContainer/reformat-gherkin), pytest, [pytest-bdd](https://pytest-bdd.readthedocs.io/en/latest/) and so on.
```

Comment:
we can add to check the python version when run the test

Reply:
Fixed. Thx.